src/components/control_bar/control_bar.tsx

```diff
@@ -0,0 +1,296 @@
+import React, { Component, HTMLAttributes, ButtonHTMLAttributes } from 'react';
+import classnames from 'classnames';
```

Comment:
```suggestion
import classNames from 'classnames';
```

Just for consistency